templates/taxi/car_detail.html

```diff
@@ -19,4 +19,15 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  <form method="post">
+    {% csrf_token %}
+    {% if request.user in car.drivers.all %}
+      <input type="hidden" name="action" value="Delete">
+      <button type="submit" class="btn btn-danger">Delete me from this car</button>
+    {% else %}
+      <input type="hidden" name="action" value="Add">
+      <button type="submit" class="btn btn-success">Assign me to this car</button>
+    {% endif %}
```

Comment:
does these buttons change anything? I believe you should add new endpoints for that action

Reply:
these buttons work and provide the set in the conditions of the task